### PR TITLE
Update gpu doc with docker test.

### PR DESCRIPTION
**Pull request by jngz-es**
Signed-off-by: Jing Zhang <jngz@amazon.com>

### Description
Update gpu doc with docker test.
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docs/model_serving_framework/GPU_support.md [hunk 1]
```diff
@@ -430,6 +430,32 @@ For the second one run this
...
+- Docker Image: nvidia/cuda:11.6.2-cudnn8-devel-ubuntu20.04
+
+Some example commands.
+
+Host:
+```
+sudo sysctl -w vm.max_map_count=262144
+docker run -it --runtime=nvidia --gpus all -p 9200:9200 nvidia/cuda:11.6.2-cudnn8-devel-ubuntu20.04 /bin/bash
+```
+Docker:
```
Comment (ylwu-amzn) [inline docs/model_serving_framework/GPU_support.md:451]: Start OpenSearch in nivida/cuda docker container

File: docs/model_serving_framework/GPU_support.md [hunk 2]
```diff
@@ -430,6 +430,32 @@ For the second one run this
...
+```
+wget https://artifacts.opensearch.org/releases/bundle/opensearch/2.5.0/opensearch-2.5.0-linux-x64.tar.gz
+tar -xvf opensearch-2.5.0-linux-x64.tar.gz
+cd opensearch-2.5.0
+bin/opensearch
```
Comment (ylwu-amzn) [inline docs/model_serving_framework/GPU_support.md:456]: Should we run `bash opensearch-tar-install.sh` first?

File: docs/model_serving_framework/GPU_support.md [hunk 3]
```diff
@@ -430,6 +430,32 @@ For the second one run this
...
 ```
 
+# 3. DOCKER
+
+Tested on AWS EC2 `g5.xlarge`, 64-bit(x86)
+
+- Amazon Linux AMI: `AWS Deep Learning Base AMI GPU CUDA 11 (Ubuntu 20.04) 20221104`
+- PyTorch: 1.12.1
+- Docker: 20.10.21
+- CUDA: 11.6
```
Comment (ylwu-amzn) [inline docs/model_serving_framework/GPU_support.md:440]: Also add cuda version after line 13, 62?

File: docs/model_serving_framework/GPU_support.md (hunk 2 above)
Comment (jngz-es) [reply docs/model_serving_framework/GPU_support.md:456]: I didn't use this script. You are right, it is simpler to use this script to start opensearch.

File: docs/model_serving_framework/GPU_support.md (hunk 3 above)
Comment (jngz-es) [reply docs/model_serving_framework/GPU_support.md:440]: sure

**Review by ylwu-amzn — APPROVED** [review]

**Review by rbhavna — APPROVED** [review]

**Pull request merged**